packages/clay-checkbox/package.json

```diff
@@ -17,7 +17,7 @@
 	],
 	"scripts": {
 		"build": "babel src --root-mode upward --out-dir lib --extensions .ts,.tsx",
-		"build:types": "tsc --project ./tsconfig.declarations.json",
+		"build:types": "tsc --project ./tsconfig.declarations.json --emitDeclarationOnly --allowJs false",
```

Comment:
I think we can just add `allowJs: false` to `tsconfig.eclarations.json` at the root level of the project. We shouldn't need to add this to each build script.

Reply:
I had forgotten that. Much better!